Change greedy search to lazy search

bennett-zhang opened the pull request:
A greedy replacement will unintentionally override the save data for other mods. This simple fix will resolve that issue.

Fixes #19

DanielNoord commented:
@bennett-zhang Could you also bump the version number so we can immediately release this?

bennett-zhang committed:
Change greedy search to lazy search

A greedy replacement will unintentionally override the save data for other mods. This simple fix will resolve that issue.

bennett-zhang commented:
@DanielNoord Done! Thanks :)
